tsflex/features/segmenter/strided_rolling.py

```diff
@@ -112,7 +112,7 @@ def __init__(
         self.data_type = func_data_type
 
         # 0. Standardize the input
-        series_list: List[pd.Series] = to_series_list(data)
+        series_list: List[pd.Series] = to_series_list(data)  # TODO: isn't it always a list of series?
```

Comment:
Indeed, normally it is, but when you want to solely play/develop with this class, it is convenient to still have this function (I assume this has not a large duration / memory overhead)

fair point though